index.html

```diff
@@ -90,7 +91,47 @@ <h2>Artwork Design</h2>
         <!-- CONTACT -->
 
             <section class="contact">
+                <div class="formcontainer"> 
+                    <div class="formimage">
+                        <img src="images/form-img.png">
+                    </div>
+                    
+                    <div class="form">
```

Comment:
It would be better to use the <form> html tag, which is semantically speaking more accurate for this element.